data/authors/rob.mdx

```diff
@@ -0,0 +1,24 @@
+---
+name: Rob
+nym: BoltTC
+avatar: /static/images/ops/rob.jpg
+occupation: Operations Manager
+company: OpenSats
+nostr: npub1n3w4uts2r4srq3adqu9trp953dflcnw7ppn72tad4htkpsckwcmqjef6um
+email: rob@opensats.org
+ops: true
+---
+
+Rob believes Bitcoin will continue to transform the world for the better. He
+initially wrote off Bitcoin as he didn't see how it could fulfill the promises
+of the white paper. Once he spent the time to better understand Bitcoin and the
+Lightning network, he couldn't help but go deeper. Rob is an experienced
+software project manager who has led projects across the globe. He hopes that
+his work at OpenSats can help progress the Bitcoin and Nostr ecosystems so that
+more people globally have access to the world's best money and the beauty of
+decentralized communication. With these tools, he believes that we can build
+stronger communities and resist those who wish to control our freedoms.
+
+Rob loves seeing the world, meeting new people, and reading. When he isn't
+learning about Bitcoin or macroeconomics, he is immersed in Japanese in the
```

Comment:
```suggestion
learning about Bitcoin or macroeconomics, he is immersed in learning Japanese in the
```

Reply:
My attempt to address this [conversation above](https://github.com/OpenSats/website/pull/167#discussion_r1376930637)